docs/user/exercises/quiz.rst

```diff
@@ -1,2 +1,440 @@
+
+            - At the end of the list, click the Add selected Questions button |AddSelectedQuestionsButton| to import all selected questions.
+
+    - **Footer**: On the creation quiz page there is a footer with the following fields:
+
+            .. figure:: quiz/footer.png
+                :align: center
+                :scale: 50
+
+        - Error messages
+
+        - Warning messages
+
+        - Cancel button
+
+        - Save button
+
+Create Drag and Drop Model Quiz Exercise
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - In the quiz exercises section click on the Create Quiz button |CreateDragAndDropQuizButton|.
+
+        .. figure:: quiz/apollon-diagrams.png
+            :align: center
+
+    - On the Apollon Diagrams page, it is possible to see the list of Apollon Diagrams and the possible actions to perform with them.
+
+        - Open
+        - Delete
+
+    - Clicking on the creation of a new Apollon Diagram button |CreateANewApollonDiagram| opens the following form:
+
+        .. figure:: quiz/Apollon-form.png
+            :align: center
+            :scale: 50
+
+        - Title: provide the title of the Drag-and-Drop Model Quiz
+
+        - Diagram Type: It is a list picker to select between several diagrams:
+
+            - Class Diagram
+
+            - Activity Diagram
+
+            - Object Diagram
+
+            - Use Case Diagram
+
+            - Communication Diagram
+
+            - Component Diagram
+
+            - Deployment Diagram
+
+            - Petri Net
+
+            - Syntax Tree
+
+            - Flowchart
+
+        - Save button
+
+        - Cancel button
+
+    - Click the save button to open the Apollon editor
+
+        .. figure:: quiz/apollon-editor.png
+            :align: center
+
+        - Title: Allows to edit the tile of the diagram.
+
+        - Crop image to selection: Allows to download the current selection.
+
+        - Download button: To download the selection.
+
+        - Generate a quiz exercise button.
+
+        - Save button.
+
+        - Modeling field: The items for modeling the diagram will be displayed here.
+
+        - Elements to Drag and Drop in the Modeling field.
+
+Import a Quiz
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - In the quiz exercises section, click on the Import a Quiz button |ImportQuizButton|.
+
+    - The list of existing quizzes will appear.
+
+        .. figure:: quiz/import-list-quizzes.png
+            :align: center
+            :scale: 50
+
+        - The searching bar: Allows to look for a specific quiz by typing its name or part of it.
+
+        - The list of quizzes: Whit their ID, title, course and indicator if they are exam questions.
+
+        - Clicking the Import button |ImportButton| opens the quiz editor with the existing questions. Here it is possible to edit all parameters such as in **Create new quiz exercise**.
```

Comment:
```suggestion
        - Clicking the Import button |ImportButton| opens the quiz editor with the existing questions. Here it is possible to edit all parameters such as in **Create new Quiz Exercise**.
```
Maybe you can also link "Create new Quiz Exercise" to the corresponding section.